docs/en/operations/table_engines/graphitemergetree.md

```diff
@@ -95,14 +112,16 @@ default
     ...
 ```
 
-**Important:** The order of patterns should be next:
+!!! note "Attention"
+    Patterns must be strictly ordered. Read about it below.
```

Comment:
* I'd just put the order list right inside attention block and remove the "below" reference
* note -> warning